From 7ddfb7b142d0fce596cdd86119030283394ef28e Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Sun, 7 Jan 2024 15:01:17 +0100
Subject: [PATCH] importlib.util

---
 pychemia/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pychemia/__init__.py b/pychemia/__init__.py
index 2717ccd..3382469 100644
--- a/pychemia/__init__.py
+++ b/pychemia/__init__.py
@@ -8,7 +8,7 @@
 
 import sys
 import logging
-import importlib
+import importlib.util
 
 from pychemia.version import version as __version__
 from pychemia.version import author as __author__
